zebra-network/src/address_book/tests/vectors.rs

```diff
@@ -110,3 +110,39 @@ fn address_book_peer_order() {
         Some(meta_addr2),
     );
 }
+
+/// Check that `reconnection_peers` skips addresses with IPs for which
+/// Zebra already has recently connected peers.
```

Comment:
```suggestion
/// Zebra already has recently connected outbound peers.
```

Reply:
It looks like the address book should be tracking inbound connections as well: https://github.com/ZcashFoundation/zebra/blob/issue6681/zebra-network/src/peer/handshake.rs#L933

Reply:
This method returns `None` for inbound connections, so the address book never finds out about them:
https://github.com/ZcashFoundation/zebra/blob/b6ba3f70319f8283feb8e7033b25f453b90ad843/zebra-network/src/peer/handshake.rs#L245-L254
https://github.com/ZcashFoundation/zebra/blob/b6ba3f70319f8283feb8e7033b25f453b90ad843/zebra-network/src/peer/handshake.rs#L263-L270

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/7029/commits/0c769b657fbe1f66dd20b7ed874ce4b529395395.